src/components/ListResources.vue

```diff
@@ -141,11 +122,16 @@ export default {
 }
 </script>
 
-<style scoped>
+<style>
 @import '../../node_modules/vue-virtual-scroller/dist/vue-virtual-scroller.css';
 
 .files-list-row-disabled {
   opacity: 0.3;
   pointer-events: none;
 }
+
+.oc-file-picker-row {
```

Comment:
Why not `height` instead of min and max with the same value? And why don't we have this in ODS? :) (Curiously asking, not judging..)